doc/whats_new.rst

```diff
@@ -83,6 +84,8 @@ Requirements
 Bug fixes
 ^^^^^^^^^
 
+- Fix setting unknown ages if no age parameter is supplied in csv in function :func:`mne_bids.report._summarize_participants_tsv` (:gh:`912`)
```

Comment:
This is documenting a change in a private function, so this is not directly user-visible and shouldn't go into the changelog. Better do something like:

```suggestion
- :func:`mne_bids. make_report` can now handle `participants.tsv` files without an `age` column, by `Simon Kern`_ (:gh:`912`)
```